src/scribe_data/cli/total.py

```diff
@@ -206,6 +206,16 @@ def get_total_lexemes(language, data_type, doPrint=True):
     else:
         data_type_qid = get_qid_by_input(data_type)
 
+    if not language_qid:
+        print(
+            "The specified language does not exist. Please update your language_metadata.json file by using:\n"
```

Comment:
We'll need to check these outputs once the updates are done :)

Reply:
Yeah, sure! Thanks @andrewtavis !